packages/ui/src/components/Listbox/SelectStyled.stories.tsx

```diff
@@ -10,34 +10,34 @@ export default {
   component: Listbox,
 }
 
-const { Option } = Listbox
-
 const options = [
   { value: 'one', label: 'one' },
   { value: 'two', label: 'two' },
   { value: 'three', label: 'three' },
   { value: 'four', label: 'four' },
 ]
 
-export const Default = (args: any) => (
+export const Default = () => (
   <Listbox label="Default listbox">
     {options.map((option) => {
       return (
         <Listbox.Option
           label={option.label}
           value={option.value}
           children={({ active, selected }: any) => {
-            // console.log('selected', selected)
-            // console.log('active', active)
-            return <span>{option.label}</span>
+            const optionStyle = {
```

Comment:
why are we not using tailwind?

Reply:
Thanks @MildTomato, no reason in particular.
Update now, kindly take a look.